### PR TITLE
fix linter settings

### PR DESCRIPTION
**Pull request by MP91**

### CODE REVIEW

**Comment by doosuu** [comment]
What about .pre-commit-config.yml?

**Comment by MP91** [comment]
> What about .pre-commit-config.yml?

simply forgot, because there is no devcontainer -> fixed

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
```
Comment (dennismeister93) [inline .devcontainer/devcontainer.json:1]: comment can be removed

File: .devcontainer/scripts/configure-proxies.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
```
Comment (dennismeister93) [inline .devcontainer/scripts/configure-proxies.sh:2]: comment

File: .devcontainer/scripts/container-set.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
```
Comment (dennismeister93) [inline .devcontainer/scripts/container-set.sh:2]: comment

File: .devcontainer/scripts/onCreateCommand.sh [hunk 4]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
```
Comment (dennismeister93) [inline .devcontainer/scripts/onCreateCommand.sh:2]: comment

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (MP91) [reply .devcontainer/devcontainer.json]: done

File: .devcontainer/scripts/configure-proxies.sh (hunk 2 above)
Comment (MP91) [reply .devcontainer/scripts/configure-proxies.sh]: done

File: .devcontainer/scripts/container-set.sh (hunk 3 above)
Comment (MP91) [reply .devcontainer/scripts/container-set.sh]: done

File: .devcontainer/scripts/onCreateCommand.sh (hunk 4 above)
Comment (MP91) [reply .devcontainer/scripts/onCreateCommand.sh]: done

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**